ansible/roles/operator-pipeline/tasks/operator-hosted-pipeline-trigger.yml

```diff
@@ -35,7 +35,7 @@
                 bindings:
                   - ref: operator-hosted-pipeline-trigger-binding
                 template:
-                  name: operator-hosted-pipeline-trigger-template
+                  ref: operator-hosted-pipeline-trigger-template
```

Comment:
`template.name` field is deprecated in favour of `template.ref`
https://github.com/tektoncd/triggers/pull/898